src/PactNet/Drivers/InteropActionExtensions.cs

```diff
@@ -19,5 +21,14 @@ public static void CheckInteropSuccess(this bool success)
                 throw new PactFailureException("Unable to perform the given action. The interop call indicated failure");
             }
         }
+
+        public static void CheckInteropSuccess(this StringResult success)
+        {
+            if (success.tag!=StringResult.Tag.StringResult_Ok)
```

Comment:
similar formatting question - based on what I see elsewhere there is usually space before/after `!=`.